src/commands/deploy/functions.ts

```diff
@@ -190,13 +190,7 @@ export default class DeployFunctions extends Command {
       cli.exit(1);
     }
     if (flags.json) {
-      cli.styledJSON({
-        status: 0,
-        result: {
-          results,
-        },
-        warnings: [],
-      });
+      return results;
     }
```

Comment:
You don't need, and in fact you don't want, the `if (flags.json)` here. Just do `return results;` without an `if` statement. These don't get displayed in the terminal output unless the user passes in `--json` but they _do_ need to be returned always for the command schema generation.

Reply:
Interesting. Ok!